src/shared-components/selectorButton/index.js

```diff
@@ -21,12 +21,18 @@ const propTypes = {
   ]),
   type: PropTypes.oneOf(['primary', 'secondary']),
   selector: PropTypes.oneOf(['radio', 'checkbox']),
+  Icon: PropTypes.node,
+  iconWidth: PropTypes.number,
+  iconHeight: PropTypes.number,
 };
 
 const defaultProps = {
-  onClick: () => {},
+  onClick: () => {
+  },
   type: 'primary',
   selector: 'radio',
+  iconWidth: 16,
+  iconHeight: 16,
 };
 
 const SelectorButton = ({
```

Comment:
Probably not a big deal but I usually try to avoid passing in capitalized props --> `Icon={...}`. 
You could reassign it in the function and skip the ternary as well for readability. Just a thought. Looks good otherwise!

```jsx
const SelectorButton = ({
  checked,
  onClick,
  children,
  type,
  selector,
  icon,
  iconWidth,
  iconHeight,
  ...rest
}) => {
  let Icon = CheckMarkIcon;
  let width = 11;
  let height = 11;

  if (icon) {
    Icon = icon;
    width = iconWidth;
    height = iconHeight;
  }

  return (
    <OuterContainer
      onClick={onClick}
      onKeyPress={onClick}
      tabIndex="0"
      selector={selector}
      {...rest}
    >
      <SelectorContainer>
        <Icon 
          css={css`
            ${selectorIcon};
            color: ${checked ? COLORS.white : 'transparent'};
          `}
          width={width}
          height={height}
        />
        <Selector type={type} checked={checked} selector={selector} />
      </SelectorContainer>

      {children && (
        <TextContainer>{children}</TextContainer>
      )}
    </OuterContainer>
  );
};
```

Reply:
@youngmanalive Same actually re: non-capitalized props; I had a recent memory of seeing a capitalized prop which made me change my mind on that, but I was actually mistaken, it was just assigned to a capitalized variable within the render method. Fixed in 23f15cd999e65e8464948b9f0efb7381f031c8e8.

Reply:
(Ternary also obviated in that same commit)

Reply:
👍